examples/data-sources/pingfederate_administrative_account/data-source.tf

```diff
@@ -0,0 +1,14 @@
+resource "pingfederate_administrative_account" "myAdministrativeAccount" {
```

Comment:
I don't think the data sources should include resource HCL in the examples. Just the data block would be fine. That's what PD and PingOne have.